scripts/generate-changelog.mjs

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env node
+
+/**
+ * Pass the pull request number and category as arguments.
+ * Get the PR title and author name via the GitHub API and create a file in ./changelog_unreleased
+ *   $ node ./scripts/generate-changelog.mjs 10961 typescript
+ */
+
+import fs from "node:fs/promises";
+import path from "node:path";
+import fetch from "node-fetch";
+import createEsmUtils from "esm-utils";
+import { CHANGELOG_CATEGORIES } from "./utils/changelog-categories.mjs";
+
+const { __dirname } = createEsmUtils(import.meta);
+
+const [prNumberString, category] = process.argv.slice(2);
```

Comment:
Can we get those by select and input? Maybe by [`Inquirer.js`](https://github.com/SBoudrias/Inquirer.js), [`enquirer`](https://github.com/enquirer/enquirer), or whatever you prefer.

Reply:
It is helpful especially for beginners. Thanks.